homeassistant/components/comelit/coordinator.py

```diff
@@ -47,3 +51,17 @@ async def _async_update_data(self) -> dict[str, Any]:
         await self.api.logout()
 
         return devices_data
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Set device info."""
+
+        return DeviceInfo(
+            identifiers={
+                (DOMAIN, self.config_entry.entry_id),
+            },
+            manufacturer="Comelit",
+            model="Serial Bridge",
+            hw_version="20003101",
```

Comment:
Are you sure this is the only possible `hw_version` even in the future?
Do you know if the API can retrieve the version? I doubt that when we leave it fixed, sooner or later, this information is not correct anymore.